openwisp_monitoring/db/backends/influxdb/client.py

```diff
@@ -146,6 +146,15 @@ def read(self, key, fields, tags, **kwargs):
             conditions = 'WHERE %s' % ' AND '.join(conditions)
             q = f'{q} {conditions}'
         if order:
+            if order == 'time':
+                order = 'time ASC'
+            elif order == '-time':
+                order = 'time DESC'
+            else:
+                raise self.client_error(
+                    f'Invalid order "{order}" passed.\nYou may pass "time" / "-time" to get '
+                    'result sorted in ascending /descending order respectively.'
```

Comment:
What  if you want to order by a different value?

I think the logic here is checking if `order` starts with`-` or not, and then constructing `{order} ASC/DESC`

Reply:
https://github.com/openwisp/openwisp-monitoring/blob/1825d7788a07be6a2afdf4eeeb1c29c8bd571e1f/openwisp_monitoring/device/tests/test_api.py#L401

Reply:
Anything other than time is not supported for `influxdb`, https://stackoverflow.com/questions/58733165/is-there-a-way-to-sort-influxdb-by-a-field-value-other-than-time.
I also didn't think there's a requirement for it, currently. Let me know if I am missing anything.

Reply:
As you pointed out in the gitter chat, influxdb only allows ordering results by time, so it's ok - https://stackoverflow.com/questions/58733165/is-there-a-way-to-sort-influxdb-by-a-field-value-other-than-time

Please add the following comment so that future developers can save a search about it ;)

```
# As of now, influxdb only allows to order by time
```